doc/versioning_and_life_cycle.md

```diff
@@ -0,0 +1,73 @@
+
+## Versioning
+
+The Azure CLI follows [Semantic Versioning](https://semver.org/) for version numbering. The CLI version consists of three parts `MAJOR.MINOR.PATCH`:
+- MAJOR version is upgraded only when there is a huge revolution, such as the core changes that will break commands' behavior globally.
+- MINOR version is upgraded for new features with the normal release process. All feature updates must be forward compatible, and breaking changes can only be published in the breaking change window *(Ignite sprint or Build sprint)*.
+- PATCH version is upgraded for hotfixes through the hotfix release process.
+
+## Life Cycle
+
+### Release States
+
+Azure CLI differs from the [three life cycles](https://docs.microsoft.com/learn/modules/describe-service-life-cycle-microsoft-365/2-private-public-general-availability) of an Azure Service.  CLI only uses two states, **GA** (Generally Available) and **Preview**, making it easier to understand and maintain.
+
+- The **GA** state means that a feature is mature, stable, available to all customers, and has formal support. A GA feature must be forward compatible and not prone to cause breaking changes.
```

Comment:
Yes, so our description is `not prone to cause breaking changes`, because it is less prone to breaking change than `Preview`. Do you think this expression is not accurate enough?